frontend/src/components/layout/Layout.module.scss

```diff
@@ -7,17 +7,23 @@
 
   // See https://fkhadra.github.io/react-toastify/how-to-style#override-css-variables
   --toastify-toast-width: 95%;
-  --toastify-color-success: #{$color-green-60};
-  --toastify-text-color-success: #{$color-white};
-  --toastify-color-error: #{$color-red-50};
-  --toastify-text-color-error: #{$color-white};
+  --toastify-color-light: #{$color-white};
+  --toastify-color-dark: #{$color-grey-50};
+  --toastify-color-info: #{$color-blue-50};
+  --toastify-color-success: #{$color-green-50};
+  --toastify-color-warning: #{$color-yellow-50};
+  --toastify-color-error: #{$color-red-60};
 }
 
 .toast {
-  border-radius: $border-radius-md;
   text-align: center;
-  font-weight: 700;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 21px;
+  border-radius: $border-radius-sm;
   font-family: $font-stack-base;
+  padding-top: 9.5px;
+  padding-bottom: 9.5px;
```

Comment:
Suggestion: Use Protocol tokens to set padding. We do not have to be pixel perfect, and we nudge designs to where they fit within the set variables. (Example: Random graytones get matched to the closest `$color-grey-##`, etc.) 

In this case: 
```suggestion
  padding-top: $spacing-md;
  padding-bottom: $spacing-md;
```

Reply:
This also brings up a larger conversation with UX: The current toast design (from Nebula) is never wider than ~400px. I would check with UX and confirm they want full width toasts, or if there's any design changes necessary for a full-bodied 2000px wide toast. 🍞

Reply:
Trying as best as possible to stick to Protocol :) - code updated.

Reply:
Also spoke to Eduardo - This is what was adviced: 
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/3924990/165650089-89d43cf3-7f8d-4870-8e7e-001dbe7c77a2.png">

Using `$content-xl` as the max-width for the toast container, you can see it goes over a bit, in comparison to the container below it (with content) - the container below it has a padding of 16px or so on each side. 

This is what it looks like if I remove `layout-sm` (32px) from the max-width value. Up to you what you want us to do. 
<img width="1038" alt="image" src="https://user-images.githubusercontent.com/3924990/165650297-15504483-f6d3-48b1-9cce-550a37cf5006.png">